src/main/java/com/google/sps/data/Game.java

```diff
@@ -28,6 +34,52 @@ public static Builder builder() {
     return gameData;
   }
 
+  public static Game buildWithJson(String gameJsonStr) {
+    // The function recieves the JSON and converts to a JsonObject
+    String gameJson = gameJsonStr;
```

Comment:
nit: is this used? do you want to just use one of these?

Reply:
Yes, this is used but I wasn't sure if I wanted to add this to `NewGameServlet.java` or to `Game.java` but I stuck with this, I think I'll change it to the other file.